Server/JokesOnYou.Web.Api/Services/JwtTokenService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using JokesOnYou.Web.Api.Models;
+using JokesOnYou.Web.Api.Services.Interfaces;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+
+namespace JokesOnYou.Web.Api.Services
+{
+    public class JwtTokenService : ITokenService
+    {
+        private readonly JwtSecurityTokenHandler _tokenHandler = new JwtSecurityTokenHandler();
+        private readonly byte[] _key = Encoding.ASCII.GetBytes("We need to use a Secret Handler here");
+
+        public string GetToken(User user)
+        {
+            return GenerateToken(user);
+        }
+
+        public bool ValidateToken(string token)
```

Comment:
What will this be used for? Since our middleware will authenticate the token

Reply:
removed